doc/man1/flux-shell.rst

```diff
@@ -320,6 +320,24 @@ options are supported by the builtin plugins of ``flux-shell``:
   this will also be passed to the invoked plugin. Normally, this option will
   be set by the ``flux mini --taskmap`` option.
 
+**stage-in**
+  Copy files to $FLUX_JOB_TMPDIR that were previously mapped using
+  :man1:`flux-filemap`.
```

Comment:
we seem to list "common" or "useful" shell options in `flux-mini`.  Do some of these pass the bar to be included there.

Reply:
Maybe a brief mention with references like this?
```diff
diff --git a/doc/man1/flux-mini.rst b/doc/man1/flux-mini.rst
index 002d26a25..4f15125d1 100644
--- a/doc/man1/flux-mini.rst
+++ b/doc/man1/flux-mini.rst
@@ -758,6 +758,9 @@ overridden in some cases:
    Configure the PMI plugin's built-in key exchange algorithm to use a
    virtual tree fanout of ``N`` for key gather/broadcast.  The default is 2.
 
+**stage-in**
+   Copy files previously mapped with :man1:`flux-filemap` to $FLUX_JOB_TMPDIR.
+   See :man1:`flux-shell` for more *stage-in* options.
 
 RESOURCES
 =========
```

Reply:
sounds good